config.py

```diff
@@ -9,3 +9,7 @@
 # In bytes, csvapi will stop downloading files if they reach this size
 # Default to 100 Mo
 MAX_FILE_SIZE = 1024 * 1024 * 100
+# Set this to an array of hosts to filter calls by referer
+# It will also match subdomains
+# e.g. REFERERS_FILTER = ['data.gouv.fr'] will match 'demo.data.gouv.fr'
```

Comment:
REFER**R**ERS_FILTER

Reply:
yep, I changed my mind in the mean time ;-)